zokrates_core/lib/gm17.cpp

```diff
@@ -8,6 +8,7 @@
 #include "gm17.hpp"
 #include "util.hpp"
 #include <cassert>
+#include <fstream>
```

Comment:
is this needed? we're trying to keep the backend from requiring a fs

Reply:
Good catch